forte/processors/data_augment/utils/utils.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Utility functions
+"""
+
+from os import path
+
+import numpy as np
+
+from texar.torch.data import Embedding, load_glove
+from texar.torch.data import Vocab
+
+__all__ = [
+    "l2_norm",
+    "load_glove_vocab"
+]
+
+
+def l2_norm(word_vecs):
+    r"""Calculate the L2 norm
+
+    Args:
+        word_vecs: 2d numpy array of word vectors.
+
+    Returns:
+        Normalized vectors.
+    """
+    norm = np.sqrt((word_vecs * word_vecs).sum(axis=1))
+    return word_vecs / norm[:, np.newaxis]
+
+
+def load_glove_vocab(glove_filename: str) -> Vocab:
```

Comment:
I think the original `Vocab` function has this method: https://github.com/asyml/texar-pytorch/blob/master/texar/torch/data/vocabulary.py#L105. Let's not re-do it